connect/README.md

```diff
@@ -1,88 +1,258 @@
+
+  decrement = () => {
+    this.count--;
+  };
+}
+
+const MyComponent = ({ value, increment, decrement }) => (
+  <div>
+    <button onClick={decrement}>-</button>
+    <input readOnly value={value} />
+    <button onClick={increment}>+</button>
+  </div>
+);
+
+export default ylem(Store, MyComponent);
 ```
-npm install ylem --save
-```
 
-* [Configure with Webpack](./docs/getting-started-webpack.md)
+---
+
+This isn't limited to a components internal state either, **ylems** observable objects and arrays are _reactive_ to each other, which allows you to move your domain models and business logic out of your components and into reactive classes and stores.
+
+Learn how to architect your React app with **ylem** for concise and powerful state management without the struggle.
+
+- [Get Started](#getting-started)
+- [API](./docs/api.md)
+- [Advanced Topics](./docs/advanced-topics.md)
+- [Examples](./docs/examples.md)
+
+## Getting Started
+
+_This small getting started guide is just to get you familiar with the concepts around **ylem**. For more specific information checkout out [API docs](./docs/api.md) and for more information and different ways to use **ylem** checkout out the [Advanced Topics](./docs/advanced-topics.md)._
+
+### Step 1
+#### Start by setting up a new app with [create-react-app](https://github.com/facebook/create-react-app)
+
+If you need information for how to set up a new project with create-react-app we've provided a quick guide:
+* [Start locally with create-react-app](./docs/getting-started-create-react-app.md)
+* [Just jump right in with CodeSandbox](https://codesandbox.io/s/j311ryqpy?module=%2Fsrc%2FApp.js)
+
+_Alternatively, if you want to start a new project using [StealJS](https://stealjs.com/) here is a guide on how to configure Steal for a **ylem** project:_
 * [Configure with StealJS](./docs/getting-started-steal.md)
 
-## Usage
+### Step 2
+#### Install ylem and modify `App.js`
+
+```sh
+npm install ylem
+```
+
+Once **ylem** is installed, modify `src/App.js` to match the example below _(copy and paste)_. Feel free to look over what is going on here, but we'll take each part in turn.
+
+```js
+import React from "react";
+import "./App.css";
+import ylem, { ObserveObject } from "ylem";
+
+var store = new ObserveObject({
+  count: 0,
+  increment() {
+    this.count++;
+  }
+});
+
+export default ylem(() => (
+  <div>
+    <input readOnly value={store.count} />
+    <button onClick={store.increment}>+</button>
+  </div>
+));
+```
 
-### Step 1 - Create observable stores using ObserveObject
+### Step 3
+#### Understanding ylems Observable Objects
 
-The recommended way for creating observable stores is to create a class which extends `ObserveObject`:
+**ylem** exports some classes `ObserveObject` and `ObserveArray`. From these classes you can construct **observable instances**, which will inform **ylems** "observers" whenever a property is changed or mutated on the instance. This is the basis for our reactive stores.
```

Comment:
I kind of wanted to at least mention it, because I think it's the first question people would have when we talk about observable objects... "does it work with arrays"